hi-ml-cpath/src/health_cpath/models/encoders.py

```diff
@@ -105,6 +106,15 @@ def __init__(self, tile_size: int, n_channels: int = 3) -> None:
         super().__init__(resnet50, tile_size, n_channels, apply_imagenet_preprocessing=False)
 
 
+class SwinTransformer_NoPreproc(ImageNetEncoder):
+    def __init__(self, tile_size: int, n_channels: int = 3) -> None:
```

Comment:
please add some docstrings explaining this encoder

Reply:
I think the name is self explanatory, we don't have docs for Resnet encoders either

Reply:
I think we should explain what version this is `swin_tiny_patch4_window7_224` what does 4, 7, tiny refer to?

Reply:
Resnets are self explanatory (with suffix 18 and 50) but there are multiple versions of swin transformers. I think the name should be `SwinT_Transformer` or something similar

Reply:
Agreed, patch 4 mean that we using a patch size 4x4 for patching the input images into small tokens window 7 is the size of the sliding window 7x7 within which attention is computed, 224 is the input image size, tiny is for the model's size.
I'll add this in docs. Thanks